Improved wording in auth.models.User field docs. (#17502)
django/django: docs/ref/contrib/auth.txt

    .. attribute:: is_staff

>         Boolean. Determines whether this user can access the admin site.

Reviewer: ```suggestion
        Boolean. Allows this user to access the admin site.
```

    .. attribute:: is_active

>         Boolean. Indicates that this user account should be considered
>         active. We recommend that you set this flag to ``False`` instead of
>         deleting accounts; that way, if your applications have any foreign keys
>         to users, the foreign keys won't break.

Reviewer: ```suggestion
        Boolean. Marks this user account as active. We recommend that you
        set this flag to ``False`` instead of deleting accounts; that way, if your
        applications have any foreign keys to users, the foreign keys won't
        break.
```

    .. attribute:: is_superuser

>         Boolean. Gives this user all permissions without explicitly

Reviewer: Yeah, I see your point. Maybe:

```suggestion
        Boolean. Treats this user as having all permissions without explicitly
```

>         Boolean. Gives this user all permissions without explicitly

Author: Much more accurate to me!

>         Boolean. Treats this user as having all permissions without explicitly
>         assigning them.

Author: ```suggestion
        Boolean. Treats this user as having all permissions without assigning any permission to it in particular.
```

Author: Because "explicitly assigning" == "assigning".

    .. attribute:: is_superuser

>         Boolean. Determines whether this user has all permissions without
>         explicitly assigning them.

Reviewer: ```suggestion
        Boolean. Gives this user all permissions without explicitly
        assigning them.
```

    .. attribute:: date_joined

>         A datetime indicating when the account was created. Is set to the
>         current date/time by default when the account is created.

Reviewer: ```suggestion
        The date/time when the account was created.
```
